### PR TITLE
Carthage support

### PR DESCRIPTION
**Pull request by hixfield**
- updated the readme with section about Carthage support
- made the WistiaKit and WistiaKitCore scheme "shared" (required for Carthage)

### CODE REVIEW

**Comment by acj** [comment]
@hixfield hey, thanks for this.

This works for me locally with a sample project. One wrinkle is that Alamofire needs to be built somehow. For my sample app, I needed to add Alamofire and AlamofireImage to my Cartfile. Not a problem, but it's something that we should mention in the readme. And while Carthage users are probably used to the workflow, it might be worth noting that the built framework(s) need to be dragged into the app project.

Carthage gives us a couple of options for building Alamofire frameworks as part of WistiaKit if we want to. For simplicity's sake, though, I'm leaning towards letting the host app handle the dependencies. It's explicit, gives the app more control, and reduces compile-time work for apps that already build Alamofire through other means. Please let us know if you have any strong feelings there.

**Comment by hixfield** [comment]
@acj happy to contribute

There is a long thread about this frameworks that use other frameworks topics here https://github.com/Carthage/Carthage/issues/844

Basically if I read and interpret it right your proposed approach would be ok (actually this is how I did it in my app as well). +1. So, we just update the readme to indicate that WistiaKit depends on Alamofire (and all other frameworks we need) and that the cartflle of the project should include them.

How shall we proceed?

File: README.md [hunk 1]
```diff
@@ -49,6 +50,21 @@ To install, simply add the following line to your Podfile:
...
 pod "WistiaKit"
 ```
+### Carthage
+Starting from `0.30.2` WistiaKit should work with Carthage, sample Cartfile:
+
+```ruby
+github "Wistia/Wistiakit" ~> 0.30.2
+```
+At this moment this creates 2 frameworks `WistiaKit` and `WistiaKitCore`.
+In the future this could be converted into one framework, but for now this is what we have, and its working!
```
Comment (spinosa) [inline README.md]: Let's explain the rationale behind the two frameworks (can simply reference the following section, "Usage", where they are explained in detail) and drop this line.

File: README.md [hunk 2]
```diff
@@ -49,6 +50,21 @@ To install, simply add the following line to your Podfile:
...
+
+Just remember to include both of these frameworks in your swift code:
```
Comment (spinosa) [inline README.md]: ...and tweak this to read "...include the _necessary_ frameworks..."

File: README.md (hunk 1 above)
Comment (hixfield) [inline README.md]: done

File: README.md (hunk 2 above)
Comment (hixfield) [inline README.md]: done

File: README.md [hunk 3]
```diff
@@ -51,22 +51,23 @@ To install, simply add the following line to your Podfile:
 pod "WistiaKit"
 ```
 ### Carthage
-Starting from `0.30.2` WistiaKit should work with Carthage, sample Cartfile:
+Starting from `0.30.2` WistiaKit should work with Carthage. Don't forget to include `Alamofire` and `AlamofireImage` as they are used by WistiaKit. Sample Cartfile:
 
 ```ruby
 github "Wistia/Wistiakit" ~> 0.30.2
```
Comment (spinosa) [inline README.md]: need a capital K: `"Wistia/WistiaKit"`

**Comment by hixfield** [comment]
@spinosa you still need to tag it 0.30.2, currently Carthage is integration (as specified on in the readme) is not working because it looks for this tag...

**Pull request merged**